students/jakevanhulzen/Lab2Part3/Lab2Part3.ino

```diff
@@ -0,0 +1,44 @@
+
+const int SONAR_PIN = A0;   // Analog pin connected to the sensor output
+const int R_LED = 11; // Red LED on pin 9
+const int G_LED = 10; // Green LED on pin 10
+const int B_LED = 9; // Blue LED on pin 11
+const float VOLTAGE_SUPPLY = 5.0; // Voltage supply to the sensor in volts
+const float VOLTAGE_PER_INCH = VOLTAGE_SUPPLY / 512.0; // Voltage per inch scaling factor
```

Comment:
Double check your data sheet... the one I found gives this value in volts per cm